src/environment_setup/conda.mli

```diff
@@ -1,13 +1,20 @@
 open Biokepi_run_environment
 
+type python_version_type = [
+  | `Python2
+  | `Python3
+  | `PythonCustom of string
+  | `PythonToolDependency of string
+]
+
```

Comment:
`+1` for the documentation

I don't mind the polymorphic variant Vs regular variant.

The guideline is to use underscores and never CamelCase (so → `` `Python_tool_dependency``)